tutorials/Dashboards_and_Low-Code_Apps/GQI/Creating_Minus_Operator.md

```diff
@@ -0,0 +1,169 @@
+---
+uid: Creating_Minus_Operator
+---
+
+# Creating a minus operator
+
+In this tutorial, you will learn how to create a custom operator that subtracts two values and adds the result to a table. You could, for example, use this operator to calculate the profit for each project in this table:
+
+| Project (String) | Customer (Int) | Sales (Double) | Cost (Double) |
+| ---------------- | -------------- | -------------- | ------------- |
+| Project 1        | Customer A     | €1000          | €750.0        |
+| Project 2        | Customer B     | €2000          | €250.0        |
+| Project 3        | Customer A     | €1500          | €500.0        |
+
+The result will look like this:
+
+| Project (String) | Customer (Int) | Sales (Double) | Cost (Double) | Profit (Double) |
+| ---------------- | -------------- | -------------- | ------------- | --------------- |
+| Project 1        | Customer A     | €1000          | €750.0        | €250.0          |
+| Project 2        | Customer B     | €2000          | €250.0        | €1750.0         |
+| Project 3        | Customer A     | €1500          | €500.0        | €1000.0         |
+
+> [!TIP]
+> See also: [Configuring a custom operator for a query](xref:GQI_Custom_Operator)
+
+> [!NOTE]
+> This tutorial uses DataMiner version 10.3.2.
+
+## Prerequisites
+
+- DataMiner version 10.2.7/10.3.0 or higher.
+
+- Depending on your DataMiner version, you may need to enable the [*GenericInterface* soft-launch option](xref:Overview_of_Soft_Launch_Options#genericinterface).
+
+  > [!NOTE]
+  > Future DataMiner versions may already include this feature. To check the release version of a soft-launch option, see [Overview of soft-launch options](xref:Overview_of_Soft_Launch_Options).
+
+  > [!TIP]
+  > See [Activating soft-launch options](xref:Activating_Soft_Launch_Options).
+
+## Create a basic implementation
+
+Create a new class that implements the `IGQIColumnOperator` and the `IGQIRowOperator` interfaces.
```

Comment:
How should they do this? Can you add more information on how to add a new class, @skyline-rge?